app/models/user.rb

```diff
@@ -492,14 +492,8 @@ def senior_delegate_for?(user)
     user.senior_delegates.include?(self)
   end
 
-  def age_in_years
-    years_since_birth = Date.today.year - dob.year
-    birthday_happened = Date.today.day >= dob.day && Date.today.month >= dob.month
-    birthday_happened ? years_since_birth : years_since_birth - 1
-  end
-
   def below_forum_age_requirement?
-    age_in_years < FORUM_AGE_REQUIREMENT
+    (Date.today - 13.years) < dob
```

Comment:
Perhaps this should be `<=` so that you can access the forums on the day of your 13th birthday

Reply:
Also, it seems you're completely ditching `FORUM_AGE_REQUIREMENT` now, is there a special reason for this? `Date.today - FORUM_AGE_REQUIREMENT.years` should :tm: work